### DISCUSSION TITLE
drag & drop, autoez issue

### DISCUSSION THREAD

**Discussion by spinney2** [jsme-editor/jsme-editor.github.io#10] [Q&A]
I am making some pages where I use one JSME editor and several depict applets. Using 2014-06-28 I can drag & drop between the editor and depict applets, but when I try to get a smiles string from the depict applet the autoez function doesn't work. I tried updating to 2020-12-26 but now the drag & drop between the JSME editor and depict applets doesn't work. Is this a limitation with the depict function? Any ideas how to fix this problem?

Thanks, Rick

**Comment by jsme-editor** [comment]
Use the options "depict,paste" to enable drag&drop and structure pasting  input in depict mode. By default, in the recent versions of JSME, the editor in depict mode does not accept pasting, 

```html
<div code="JME.class" archive="JME.jar" width=340 height=340>
    <param name="options" value="depict,paste">
</div>
```
```javascript
        //this function will be called after the JavaScriptApplet code has been loaded.
        function jsmeOnLoad() {
            jsmeApplet = new JSApplet.JSME("jsme_container", "180px", "140px", {
                "options": "depict,paste"
            });
        }
```

**Comment by spinney2** [comment]
Thanks, that fixed the drag and drop issue. I am using     "options" : "depict,paste,polarnitro,autoez,nostereo" as I need to get the stereochemistry of double bonds at this point. The smiles is generated but without the autoez function. Is there a way to enforce autoez in the depict mode?

Thanks, Rick

Dr. Richard Spinney
Department of Chemistry & Biochemistry
The Ohio State University
120A Celeste Laboratory
120 West 18th Ave
Columbus, Ohio 43210

Email: ***@***.******@***.***>
Phone:    (614) 247 - 6847
Fax:          (614) 292 - 1685

**Comment by jsme-editor** [comment]
> Is there a way to enforce autoez in the depict mode?
>I am using     "options" : "depict,paste,polarnitro,autoez,nostereo"

Yes. Do not use the nostereo option. Using nostereo  disables all stereo information including the double bond stereochemistry.

 "options" : "depict,paste,polarnitro" should work because the options autoez and stereo are enabled by default.
